flake/colmena.nix

```diff
@@ -111,6 +111,17 @@ in
           "-M${toString (cfg.totalMaxHeapSizeMiB / cfg.instances)}M"
         ];
       });
+
+      # Tracing
+      tracers = {
+        services.cardano-node.extraNodeConfig.LocalTxMonitorProtocol = true;
+        services.cardano-node.extraNodeConfig.options = {
+          mapSeverity = {
+            cardano.node.LocalTxMonitorProtocol = "Debug";
```

Comment:
```suggestion
            "cardano.node.LocalTxMonitorProtocol" = "Debug";
```

Reply:
Maybe this will render to `"cardano.node.LocalTxMonitorProtocol": "Debug"`

Reply:
It does!